run/rclone_archive/archive-clips.sh

```diff
@@ -2,6 +2,6 @@
 
 while [ -n "${1+x}" ]
 do
-  rclone --config /root/.config/rclone/rclone.conf move -L --transfers=1 --files-from "$2" "$1" "$RCLONE_DRIVE:$RCLONE_PATH" >> "$LOG_FILE" 2>&1
+  rclone --config /root/.config/rclone/rclone.conf move -L $RCLONE_FLAGS --transfers=1 --files-from "$2" "$1" "$RCLONE_DRIVE:$RCLONE_PATH" >> "$LOG_FILE" 2>&1
```

Comment:
It would be good to reference this as an array, so that multiple options can be specified.
In other words, reference it as `"${RCLONE_FLAGS[@]:-}"`
Then in the config file you can either specify a single option as usual, or multiple options like `export RCLONE_FLAGS=(--flag1 --flag2)`